Project.toml

```diff
@@ -59,9 +60,10 @@ Cairo = "159f3aea-2a34-519c-b102-8c37f9878175"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 DelimitedFiles = "8bb1440f-4735-579b-a4ab-409b98df4dab"
 LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
+MarketData = "945b72a4-3b13-509d-9b46-1525bb5c06de"
```

Comment:
people already complain about Gadfly's huge dependency tree.  how about moving MarketData into docs/Project.toml, and then re-writing the test to use a short vector of hard-coded values?

Reply:
I put it in the `[extra]` section of `Project.toml`, since I need it for the testing cases only. Is `extra` section okay?